static/app/views/performance/transactionDetails/eventMetas.tsx

```diff
@@ -186,6 +189,29 @@ const QuickTraceContainer = styled('div')`
   }
 `;
 
+function EventID({event}: {event: Event}) {
+  return (
+    <EventIDContainer>
+      <EventIDWrapper>{getShortEventId(event.eventID)}</EventIDWrapper>
```

Comment:
What if we made the entire ID copyable? You could still have the icon to indicate that it is.